gem/lib/rb_sys/extensiontask.rb

```diff
@@ -45,6 +45,7 @@ def init(name = nil, gem_spec = nil)
         gemspec.files.reject! { |f| f.end_with?(".rs") }
         gemspec.files.reject! { |f| f.match?(/Cargo.(toml|lock)$/) }
         gemspec.files.reject! { |f| extconf.end_with?(f) }
+        gemspec.extensions.reject! { |f| f.end_with?("Cargo.toml") }
```

Comment:
Should we also consider `Cargo.lock` here?

Reply:
I think this is good, since rubygems will deny any Cargo.lock anyway.